src/components/nutrition/label/NutritionLabel.tsx

```diff
@@ -0,0 +1,172 @@
+import {styled, Box, Typography} from "@mui/material";
+
+import SeparatorBar from "./SeparatorBar"
+import NutrientRow from "./NutrientRow"
+
+const NUTRIENT_INDENT = "4pt";
+
+interface LabelContainerProps {
+    backgroundColor?: string,
+    width?: string,
+    color?: string
+}
+
+const LabelContainer = styled(Box)<LabelContainerProps>(({ backgroundColor, width, color}) => ({
+    backgroundColor: backgroundColor || "white",
+    color: color || "black",
+    fontFamily: ["Helvetica"],
+    padding: "3pt",
+    textAlign: "left",
+    width: width || "320px",
+    margin: "auto"
+}));
+
+const Title = styled(Typography)(() => ({
+    fontSize: "21pt",
```

Comment:
pt is fairly unusual to use on web, as noted in https://css-tricks.com/css-font-size/ . Prefer using px (or rem which is better if the user has custom font sizes, but that gets trickier because you have to do the math: 1rem = 16px)